platform/darwin/src/NSDate+MGLAdditions.h

```diff
@@ -7,4 +7,7 @@
 /// Converts from a duration in seconds to a duration object usable in mbgl.
 mbgl::Duration MGLDurationInSeconds(NSTimeInterval duration);
 
+/// Converts from an mbgl duration object to a duration in seconds.
+NSTimeInterval MGLSecondsFromDuration(mbgl::Duration duration);
```

Comment:
Then should we not also rename `MGLDurationInSeconds()` to `MGLDurationFromTimeInterval()`?